app/helpers/header-date.js

```diff
@@ -1,9 +1,21 @@
 import Helper from '@ember/component/helper';
 import moment from 'moment';
 
+const locales12Hours = new Set(['en', 'bn', 'hi', 'id', 'ja', 'run', 'th', 'vi', 'ko']);
+
 export function headerDate(params) {
```

Comment:
If header date is not being used, please remove

Reply:
Used in various areaa
![Screenshot from 2020-11-26 23-49-52](https://user-images.githubusercontent.com/71120226/100382806-21531580-3042-11eb-9a0a-97bdb1c51f67.png)

Reply:
OK, please change these to general date as well